index.bs

```diff
@@ -35,37 +36,140 @@ Markup Shorthands: css no, markdown yes
+### Detecting when screen properties change ### {#usage-overview-property-changes}
 <!-- ====================================================================== -->
 
-Issue: Write this section.
+Without even considering multiple screens, sites may want to adapt to screen attribute changes. To avoid polling, `change` events are fired at the {{Screen}} object:
+
+```js
+screen.addEventListener('change') e => {
+  /* display properties have changed */
+});
+```
+
+
+<!-- ====================================================================== -->
+### Detecting the presence of extended screen areas ### {#usage-overview-extended}
+<!-- ====================================================================== -->
+
+The most basic question developers may ask to support multi-screen devices is: "Does this device have multiple screens that may be used for window placement?" This is provided by the {{Screen/isExtended}} boolean, exposed to secure contexts without a permission prompt.
+
+```js
+if (screen.isExtended) {
+  /* enable multi-screen options */
+}
+```
+
+<!-- ====================================================================== -->
+### Requesting extended screen information ### {#usage-overview-screen-information}
+<!-- ====================================================================== -->
+
+After determining that there are multiple screens, information can be requested about them using the {{Window/getScreens()}} method. This method may prompt the user for permission. The resulting object lets developers enumerate screens, inspect screen properties, and watch for changes.
+
+```js
+try {
+  const screensInfo = await window.getScreens();
+
+  // Do something with the screens right now.
+  processScreens(screensInfo);
+
+  // And also if the screens change in the future.
+  screensInfo.onscreenschange = e => {
+    processScreens(screensInfo);
+  };
+} catch (ex) {
+  /* permission denied, or other error */
+}
+
+function processScreens(screensInfo) {
+  // Build a UI listing screens.
+  clearScreenList();
+  screensInfo.screens.forEach(screen => {
+    addToScreenList({id: screen.id, name: screen.label, screen: screen});
+  });
+
+  // Set the selection to the current screen.
+  selectCurrentInScreenList(screensInfo.currentScreen.id);
+}
+```
+
+
+<!-- ====================================================================== -->
+### Showing elements fullscreen on a specific screen ### {#usage-overview-fullscreen-on-specific-screen}
+<!-- ====================================================================== -->
+
+A common use case is to present a slideshow or other media fullscreen on a particular screen, while showing controls for the content on another screen. Once a screen has been selected, either interactively, using the screen's properties, or remembering a previous selection, it can be passed to the {{Element/requestFullscreen()}} method.
+
+
+```js
+slideshowElement.requestFullscreen({ screen: await getScreenForSlideshow() });
```

Comment:
Is it worth somehow indicating that `screen` here came from the window.getScreens().screens array? (Or what type it is?)

Reply:
Yeah, I'll expand this.